app/cafe.py

```diff
@@ -0,0 +1,25 @@
+import datetime
+from app.errors import (NotVaccinatedError,
+                        OutdatedVaccineError,
+                        NotWearingMaskError)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        vaccine_info = visitor.get("vaccine")

```

Comment:
You can simplify the code by checking the vaccine status directly without creating a separate variable